.github/workflows/build-push.yaml

```
@@ -317,6 +317,20 @@ jobs:
           #   builder_image_tag_arg: "" # unused
           #   image_tag_arg: "bullseye" # latest version of rust
           #   variants: "linux/amd64,linux/arm64"
+          - name: archetect
+            dockerfile: 2-archetect/bullseye/Dockerfile
+            context: 2-archetect/bullseye
+            version_tag: "latest"
+            builder_image_tag_arg: "" # unused
+            image_tag_arg: "" # unused
+            variants: "linux/amd64,linux/arm64"
+          - name: archetect
+            dockerfile: 2-archetect/bullseye/Dockerfile
+            context: 2-archetect/bullseye
+            version_tag: "2"
+            builder_image_tag_arg: "v2.0.0"
```

```suggestion
            builder_image_tag_arg: "v2.0.7"
```

this container is built weekly, it'd be really nice to do:

```
builder_image_tag_arg: "v2"
```

and do a fetch to figure out that "v2" (as of today) is "v2.0.7"